Classes/gitx_askpasswd_main.m

```diff
@@ -45,7 +45,7 @@ -(NSPanel*)	passwordPanel
 													  backing: NSBackingStoreBuffered defer: NO];
 		[mPasswordPanel setHidesOnDeactivate: NO];
 		[mPasswordPanel setLevel: NSFloatingWindowLevel];
-		[mPasswordPanel setTitle: @"GitX SSH Remote Login"];
+		[mPasswordPanel setTitle: NSLocalizedString(@"GitX SSH Remote Login", @"Title for password panel in command line tool")];
```

Comment:
[It's not impossible][1], but I'm not sure how that would hook into NSLocalizedString.

[1]: https://github.com/juri/flue#localization